.github/workflows/README.md

```diff
@@ -12,20 +12,12 @@ This command creates custom jars that we can share with customers.
 - **description** - a description that will show on the summary, so we know the contents of that custom jar.
 
 
-### Z - Sandbox job
```

Comment:
This sandbox job is not really necessary. There is a "hack" to get workflows to show.